tests/Tests/600__ai_apps/600__anaconda_commercial_edition/600__anaconda_commercial_edition.robot

```diff
@@ -99,3 +100,8 @@ Anaconda Commercial Edition Suite Setup
   [Documentation]  Setup for ACE test suite
   Set Library Search Order  SeleniumLibrary
   RHOSi Setup
+
+Verify Anaconda In Kfdef
+    [Documentation]  Verifies if Anaconda is present in Kfdef
+    ${application_names}=  Run  oc get kfdef -n redhat-ods-applications -o json | jq ' .items[0] | select(.metadata.name="opendatahub") | [.spec.applications[].name]'
```

Comment:
could you pls use the openshiftlibrary instead of `Run oc ...` ?

Reply:
Updated.